WTWSMS/history/titles/c_faro.txt

```diff
@@ -9,6 +9,10 @@
 510.1.1={
 	holder=8622
 }
+554.1.1={
+    holder=36553 # Opimius Gaius Comenciolus 
+	liege = "e_byzantium"
+}
 570.1.1={
```

Comment:
Same as Cordoba and Aracena.

Reply:
The maps show that this county was part of Spania. There weren't any rebels or anything here.

Reply:
The current setup does not include any rebels here, but as far as I see no map indicates this being in Spania.

Reply:
https://es.wikipedia.org/wiki/Provincia_de_Spania#/media/File:Hispania_560_AD_SVG.svg 
Again, I think it enters in the c_faro.

Reply:
Disagreed, Faro is the most western province in southern Iberia.

Reply:
No, the most western one is Silves/Cilpes

Reply:
True, I confused the two. With that in mind, the borders don't correspond, so the most interesting question would be if the major cities of that province were in Spania or not.

Reply:
The city of Faro, called in roman times Ossonoba, and so, the city that names the province was actually under Spania. According to Fernando García Cortazar, in his History Atlas of Spain (Atlas de Historia de España. Spania extended until the city of Portimao, which is far more to the left than Faro.

Reply:
Good, then this is justified and will be the best option from the evidence we have.